src/actions/vstudio/vs2005_csproj.lua

```diff
@@ -314,7 +319,61 @@
 				_x(2,'<Reference Include="%s" />', name)
 			end
 		end)
+	end
+
+	function cs2005.nuGetReferences(prj)
+		if _ACTION >= "vs2010" and prj.nuget then
```

Comment:
Minor nitpick for future reference: the baking system will ensure `prj.nuget` always has a value. It will just be an empty list `{}` if not set by the user's script.